# HG changeset patch
# User Mike Miller <mtmiller@octave.org>
# Date 1521071325 25200
#      Wed Mar 14 16:48:45 2018 -0700
# Branch stable
# Node ID 9c96c084e7444705ecda6fe3af0779e160ba3ce1
# Parent  553c51dcd70e1b69c1ebcb627be7fa218d266a2d
maint: remove merge clutter from NEWS file (bug #53345)

diff --git a/NEWS b/NEWS
--- a/NEWS
+++ b/NEWS
@@ -728,20 +728,17 @@ Documentation bugs fixed:
  ** small fixes to docstrings (bug #49733)
 
  ** change text describing demo plots to reflect new ColorOrder (bug #49288)
 
 Other bugs fixed:
 
  ** add missing classdef test files (bug #49819)
 
-<<<<<<< working copy
-=======
 
->>>>>>> merge rev
 Summary of important user-visible changes for version 4.2 (2016-11-13):
 ----------------------------------------------------------------------
 
  ** The parser has been extended to accept, but ignore, underscore
     characters in numbers.  This facilitates writing more legible code
     by using '_' as a thousands separator or to group nibbles into bytes
     in hex constants.
 
